### ISSUE TITLE
WCAG 2.1: 'Gear' icon's focus indicator has insufficient contrast against its background

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#760]
## Describe the bug

This issue is related to [WCAG SC 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast) and fails [F78](https://www.w3.org/WAI/WCAG21/Techniques/failures/F78).

**To Reproduce**

1. Run assessment on any page
2. Using keyboard navigate to 'Gear' icon on the header in details view
3. Observe that the focus indicator is barely visible

Focus indicator color:- box-shadow: ```rgb(133, 133, 133)``` / ```#858585```
Background color:- ```var(--ada-brand-color);``` / ```#004880```
    
## Expected behavior

The contrast ratio of the focus indicator against the background should be at least 3:1.

## Screenshots

- Bug
![image](https://user-images.githubusercontent.com/22068434/59229523-7752a580-8b8f-11e9-8661-567a3cf931c9.png)

- Color contrast
![image](https://user-images.githubusercontent.com/22068434/59229433-3eb2cc00-8b8f-11e9-8680-7bb69b84da2c.png)

## Extension

- Accessibility Insights for Web - prod
- Browser Version: Chrome 75.0.3770.80
- Target Page: Any page

## Did you search for similar existing issues?
Yes

**Comment by ferBonnin** [comment]
@cheade could you take a look and suggest color?

**Comment by cheade** [comment]
Adding in my suggestion for focus treatment on the gear icon. It is #FFFFFF, with a dash gap of (2, 4). I've given some breathing room of 3px between the focus border and the icon itself. 

![image](https://user-images.githubusercontent.com/40274682/59473287-acacfc80-8df6-11e9-979f-bb8bf2df4efb.png)

In exploring solutions for this, I'd like to move to make sure all of our focus states are similar, so I've also updated our button state focus in the [Figma style guide](https://www.figma.com/file/Wj4Ggf6GGQBQkiDIaHfXRX2B/Accessibility-Insights-Style-guide?node-id=1%3A27)

**Comment by iamrafan** [comment]
Validated the fix in prod version 2.15.0
